sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -611,7 +611,7 @@ class HiveDDLSuite
     )
   }
 
-  test("add/drop partitions - external table") {
+  test("SEX add/drop partitions - external table") {
```

Comment:
revert the changes.

Reply:
Uh? I wonder why this typo include here lol. Just reverted changes